node/src/node/slave/machine.cpp

```diff
@@ -50,13 +50,24 @@ using detail::service::node::slave::state::inactive_t;
 using cocaine::detail::service::node::slave::stats_t;
 using cocaine::service::node::slave::id_t;
 
+namespace {
+  // TODO: move to some config with defaults?
+  const auto DEFAULT_EWMA_DURATION_5s = std::chrono::seconds(5);
+  const auto DURATION_10s = std::chrono::seconds(10);
```

Comment:
Introducing such constants without domain-specific naming is needless. Moreover `std::chrono::seconds(10)` looks better in any case.